interfaces.go

```diff
@@ -254,6 +257,24 @@ type Subnet interface {
 	DNSServers() []string
 }
 
+// StaticRoute defines an explicit route that users have requested to be added
+// for a given subnet.
+type StaticRoute interface {
+	// Source is the subnet that should have the route configured. (Machines
+	// inside Source should use GatewayIP to reach Destination addresses.)
+	Source() Subnet
+	// Destination is the subnet that a machine wants to send packets to. We
+	// want to configure a route to that subnet via GatewayIP
+	Destination() Subnet
+	// GatewayIP is the IPAddress of the
+	GatewayIP() string
+	// Metric is the routing metric that determines whether this route will
+	// take precedence over similar routes (there may be a route for 10/8, but
+	// also a more concrete route for 10.0/16 that should take precedence if it
+	// applies.) Metric should be a non-negative integer.
```

Comment:
IMO, 2 reasons
1) config.Schema doesn't support Uint
2) It allows you to have an "invalid" value inline. I don't know if that will actually be used, but "ID" is also actually a uint on the MAAS side, but is always presented as an 'int', so I followed suit.